cmd/kubeadm/app/cmd/reset.go

```diff
@@ -105,14 +113,21 @@ func (r *Reset) Run(out io.Writer) error {
 		fmt.Printf("[reset] Failed to unmount mounted directories in /var/lib/kubelet: %s\n", string(umountOutputBytes))
 	}
 
-	dockerCheck := preflight.ServiceCheck{Service: "docker", CheckIfActive: true}
-	if _, errors := dockerCheck.Check(); len(errors) == 0 {
-		fmt.Println("[reset] Removing kubernetes-managed containers.")
-		if err := exec.Command("sh", "-c", "docker ps -a --filter name=k8s_ -q | xargs -r docker rm --force --volumes").Run(); err != nil {
-			fmt.Println("[reset] Failed to stop the running containers.")
+	fmt.Println("[reset] Removing kubernetes-managed containers.")
+	crictlPath, err := exec.LookPath("crictl")
+	if err == nil {
+		if r.criSocketPath != "" {
+			fmt.Println("[reset] Cleaning up running containers using crictl with socket %s", r.criSocketPath)
+			if err := exec.Command("sh", "-c", crictlPath+" -r "+r.criSocketPath+" sandboxes --quiet | xargs -r "+crictlPath+" -r "+r.criSocketPath+" rms"); err != nil {
```

Comment:
for readability, can you please create a separate string for this and use `fmt.Sprintf`?